src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/changelist/changes/benificialowner/psc/CorporateBeneficialOwnerPsc.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.changes.benificialowner.psc;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Objects;
+import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.commonmodels.CompanyIdentification;
+
+public class CorporateBeneficialOwnerPsc extends Psc {
```

Comment:
Do we need to the `registeredOffice` property?

Reply:
Same as this comment https://github.com/companieshouse/overseas-entities-api/pull/247#discussion_r1212808080